app/views/admin/editions/locked.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_title @edition.title, @edition.format_name %>
+<% page_class "edition-show" %>
+<% initialise_script "GOVUK.adminEditionShow" %>
+
+<div class="row" data-module="track-button-click" data-track-category="button-pressed" data-track-action="<%= @edition.format_name.parameterize(separator: '-') %>-button">
+  <div class="col-md-8">
+    <div class="jumbotron">
+      <h1 class="jumbotron-header">Moved to Content Publisher</h1>
+      <p>This document has been moved from Whitehall to Content Publisher.</p>
+      <p>Information about it in Whitehall may now be out of date.</p>
+      <p>You can now edit this document in Content Publisher.</p>
+      <%= content_publisher_button(@edition) %>
+    </div>
+  </div>
+  <div class="<%= @edition.imported? ? "col-md-6" : "col-md-8" %> edition-metadata">
```

Comment:
I assume we don't care about this imported question in this view?

Reply:
I'm not really sure about what imported means but I figured I should keep it this code as close to the original as possible in case it's important for a future migrated document.